cdap-master/src/main/java/co/cask/cdap/data/tools/Main.java

```diff
@@ -84,7 +112,105 @@ private String getDescription() {
     }
   }
 
-  public void doMain(String[] args) throws Exception {
+  public Main() {
+    cConf = CConfiguration.create();
+    hConf = HBaseConfiguration.create();
+
+    Injector injector = Guice.createInjector(
+      new ConfigModule(cConf, hConf),
+      new LocationRuntimeModule().getDistributedModules(),
+      new ZKClientModule(),
+      new DiscoveryRuntimeModule().getDistributedModules(),
+      new AbstractModule() {
+        @Override
+        protected void configure() {
+          bind(HBaseTableUtil.class).toProvider(HBaseTableUtilFactory.class);
```

Comment:
I'm not sure how all our guice modules are structured, but how come we have to do all these custom bindings?  Existing modules don't have what you need?